### PR TITLE
Makes MDs/Paramedics/CMOs shout "Clear!" before defibbing

### PR DESCRIPTION
**Pull request by MrHorizons**
The title says it all, really. While bowlofcereal suggested, in the Discord, to actually have it shock people besides the user and target within a one-tile radius, I've decided the "Clear!" is enough for now.

If y'all're wondering, I've made sure to tweak the various time values such that it takes just about the same amount of time as it did before.

#### Changelog

:cl:  
rscadd: Medical doctors, paramedics, and CMOs now shout "clear!" just before defibrillating someone.
/:cl:

### CODE REVIEW

**Comment by Darkstick** [comment]
really not a fan of forcing a character to say something. It always comes off as weird and takes me out of the moment a little.

**Comment by athena413** [comment]
making a character say something bad but if this does get merged please make it so that only medical personnel will do it, if a tider comes in and starts defibbing they dont say it

**Comment by MrHorizons** [comment]
> making a character say something bad but if this does get merged please make it so that only medical personnel will do it, if a tider comes in and starts defibbing they dont say it

That does sound better. Though I suspect I'll have to do a bit more than just changing `defib.dm`, so I'll close this.

**Comment by MrHorizons** [comment]
There we go. Only "doctors" shout it before defibbing, which is defined as the user's job being either "Medical Doctor", "Paramedic", or "Chief Medical Officer".

File: code/game/objects/items/defib.dm [hunk 1]
```diff
@@ -550,7 +550,12 @@
 		var/total_brute	= 0
 		var/tplus = world.time - H.timeofdeath	//length of time spent dead
 		var/obj/item/organ/heart = H.getorgan(/obj/item/organ/heart)
-		if(do_after(user, 20, target = H)) //placed on chest and short delay to shock for dramatic effect, revive time is 5sec total
+		if(do_after(user, 15, target = H))
+			if(user.job == "Medical Doctor" || user.job == "Paramedic" || user.job == "Chief Medical Officer")
+				user.say("Clear!")
```
Comment (Amelia0010) [inline code/game/objects/items/defib.dm]:
```suggestion
				user.say("Clear!", forced = "defib")
```

**Review by Amelia0010 — DISMISSED** [review]

File: code/game/objects/items/defib.dm [hunk 2]
```diff
@@ -550,7 +550,12 @@
 		var/total_brute	= 0
 		var/tplus = world.time - H.timeofdeath	//length of time spent dead
 		var/obj/item/organ/heart = H.getorgan(/obj/item/organ/heart)
-		if(do_after(user, 20, target = H)) //placed on chest and short delay to shock for dramatic effect, revive time is 5sec total
+		if(do_after(user, 15, target = H))
+			if(user.job == "Medical Doctor" || user.job == "Paramedic" || user.job == "Chief Medical Officer")
+				user.say("Clear!", forced = "defib")
+			else
+				return
+		if(do_after(user, 5, target = H)) //Counting the delay for "Clear", revive time is 5sec total
```
Comment (JamieD1) [inline code/game/objects/items/defib.dm]: Wont this cancel the defib if they are not a medical doctor or paramedic?

Unintended?

**Review by JamieD1 — CHANGES_REQUESTED** [review]
Wont this cancel the defib if they are not a medical doctor or paramedic?

Unintended?

File: code/game/objects/items/defib.dm [hunk 3]
```diff
@@ -550,7 +550,12 @@
 		var/total_brute	= 0
 		var/tplus = world.time - H.timeofdeath	//length of time spent dead
 		var/obj/item/organ/heart = H.getorgan(/obj/item/organ/heart)
-		if(do_after(user, 20, target = H)) //placed on chest and short delay to shock for dramatic effect, revive time is 5sec total
+		if(do_after(user, 15, target = H))
+			if(user.job == "Medical Doctor" || user.job == "Paramedic" || user.job == "Chief Medical Officer")
+				user.say("Clear!", forced = "defib")
+			else
+				return
```
Comment (jcatjmeow) [inline code/game/objects/items/defib.dm]:
```suggestion
```
The return here will end the entire proc early. Not what you want.

With this change, if the if condition is not fulfilled, the next code will run, which is what we want.

**Review by jcatjmeow — APPROVED** [review]

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**